src/xml/report.html.xsl

```diff
@@ -1,62 +1,122 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 
-<xsl:stylesheet 
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
   version="1.0"
   >
-  <xsl:output method="html"/> 
- 
+  <xsl:output method="html"/>
+
   <xsl:template match="/OWASPZAPReport">
 <html>
 <head>
 <!-- ZAP: rebrand -->
 <title>ZAP Scanning Report</title>
-
+<style>
+body{
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    color: #000;

```

Comment:
This could be indented at the same level as the above (same for other following properties).